Missing dependencies

error on startup:
```
continuum-auth_1        | 
continuum-auth_1        | src/index.spec.ts(1,1): error TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
continuum-auth_1        | src/index.spec.ts(2,3): error TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
continuum-auth_1        | src/index.spec.ts(3,5): error TS2582: Cannot find name 'it'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
continuum-auth_1        | src/index.spec.ts(4,7): error TS2304: Cannot find name 'expect'.

```

--- a/continuum-auth/package.json
+++ b/continuum-auth/package.json
@@ -21,6 +21,7 @@
     "@types/express": "^4.17.1",
     "@types/jsonwebtoken": "^8.3.3",
     "@types/node-fetch": "^2.5.0",
+    "@types/jest": "^24.0.18",
     "@types/pino": "^5.8.10",
     "@types/uuid": "^3.4.5",
     "dotenv-cli": "^2.0.1",

